README.md

```diff
@@ -2451,11 +2485,28 @@ const starkbank = require('starkbank');
 })();
 ```
 
-## Merchant Purchase
+## Create a MerchantPurchase
+
+The Merchant Purchase resource can be used to charge customers with credit or debit cards. If a card hasn't been used before, a Merchant Session Purchase must be created and approved with that specific card before it can be used directly in a Merchant Purchase.
```

Comment:
```suggestion
The Merchant Purchase resource can be used to charge customers with credit or debit cards.
If a card hasn't been used before, a Merchant Session Purchase must be created and approved with that specific card before it can be used directly in a Merchant Purchase.
```